File: pyjl/tests/tests_new-py2many/plus_test.jl
function plus_test(x, y)
return x + y
end

function plus_test(x::String, y::String)::String
return x * y
end

function main()
x = "ss"
y = "zz"
res = plus_test(x, y)
@assert(res == "sszz")
println("OK");
end

main()

File: pyjl/tests/tests_new/plus_test.py
# Runtime information
def plus_test(x, y):
    return x + y

# Compile time information
def plus_test(x:str, y:str):
    return x + y

if __name__ == "__main__":
    x = "ss"
    y = "zz"
    res = plus_test(x, y)
    assert res == "sszz"
    print("OK")